integration/src/main/java/com/aws/greengrass/mqttbroker/MQTTService.java

```diff
@@ -45,6 +46,7 @@
     private final ClientDeviceTrustManager clientDeviceTrustManager;
     private final ClientDeviceAuthorizer clientDeviceAuthorizer;
     private final List<InterceptHandler> interceptHandlers;
+    private final Consumer<X509Certificate> updateServerCertificateCb = this::updateServerCertificate;
```

Comment:
Can you add a comment indicating why this is being done? I don't think it's obvious for someone without the background